pkg/apis/kops/networking.go

```diff
@@ -195,80 +195,224 @@ const CiliumIpamEni = "eni"
 
 // CiliumNetworkingSpec declares that we want Cilium networking
 type CiliumNetworkingSpec struct {
+	// Version is the version of the Cilium agent and the Cilium Operator.
 	Version string `json:"version,omitempty"`
 
-	AccessLog                string            `json:"accessLog,omitempty"`
-	AgentLabels              []string          `json:"agentLabels,omitempty"`
-	AgentPrometheusPort      int               `json:"agentPrometheusPort,omitempty"`
-	AllowLocalhost           string            `json:"allowLocalhost,omitempty"`
-	AutoIpv6NodeRoutes       bool              `json:"autoIpv6NodeRoutes,omitempty"`
-	BPFRoot                  string            `json:"bpfRoot,omitempty"`
-	ContainerRuntime         []string          `json:"containerRuntime,omitempty"`
+	// AccessLog has not been implemented in the new templates and may be removed in the future.
+	// Setting this has currently no effect.
```

Comment:
```suggestion
	// AccessLog is not implemented and may be removed in the future.
	// Setting this has no effect.
```

(and in occurrences below)